docs/components/FormatFile/Mobile.stories.tsx

```diff
@@ -14,9 +15,9 @@ export default {
 } as ComponentMeta<typeof FormatFile>;
 
 const BasicTemplate: ComponentStory<typeof FormatFile> = args => (
-  <div style={{ width: 290 }}>
+  <View style={{ flex: 1 }}>
```

Comment:
you could probably just remove this. flex 1 would make it full width which the formatfile does it be default

Reply:
@darryltec I will take a look, right now dealing with the Toast lint error

Reply:
@darryltec updated it